modules/storage-expanding-csi-volumes.adoc

```diff
@@ -20,7 +20,14 @@ CSI volume expansion does not support the following:
 
 * Dynamic provisioning is used.
 
+ifndef::microshift[]
 * The controlling `StorageClass` object has `allowVolumeExpansion` set to `true` (see xref:../storage/expanding-persistent-volumes.adoc#add-volume-expansion_expanding-persistent-volumes[Enabling volume expansion support]).
+endif::microshift[]
+
+ifdef::microshift[]
+* The controlling `StorageClass` object has `allowVolumeExpansion` set to `true` (see xref:../microshift_storage/expanding-persistent-volumes-microshift.html#add-volume-expansion_expanding-persistent-volumes-microshift[Enabling volume expansion support]).
```

Comment:
This is using an xref in the module. I know it was already like that for the regular storage's use, but we shouldn't add another one here.

I'd recommend either just removing the xref from your line here. Or if you want to fix it right, I'd remove it from both places, and add an additional resources section after the module inclusion in the two assemblies with the appropriate xrefs.